score/score.go

```diff
@@ -432,6 +432,5 @@ func (su *ScoreUtil) GetControlComplianceScore(ctrl reportsummary.IControlSummar
 	if ctrl.GetStatus().IsPassed() {
```

Comment:
Since I cant comment on the code that was not changed, I have the following questions:
1. Why do we iterate over the passed resources and won't simply use the `len(resourcesIDs.Passed())`?
2. The "if" statement: `if ctrl.GetStatus().IsPassed() {` should happen before all of the interactions

Reply:
1. I followed the implementation of the current score calculation. I assumed [this line](https://github.com/kubescape/opa-utils/blob/7e1278579ab16fd631112975caf24c924c440817/score/score.go#L311) was added after a bug had occured so I follewed suit. If you think it's redundant I can remove it.
2. Done

Reply:
1. Lets use the `len` function, please verify it works properly.
2. Tnx